src/Tests/Microsoft.Diagnostics.Monitoring.StartupHook.UnitTests/Exceptions/Eventing/ExceptionsEventSourceTests.cs

```diff
@@ -77,14 +77,17 @@ public void ExceptionsEventSource_WriteException_Event(ulong id, string message,
             listener.EnableEvents(source, EventLevel.Informational);
 
             ulong[] frameIds = frameIdsString.Split(',', StringSplitOptions.RemoveEmptyEntries).Select(ulong.Parse).ToArray();
-            source.ExceptionInstance(id, message, frameIds);
+            DateTime timestamp = DateTime.UtcNow;
```

Comment:
I think using the actual time makes sense for now - eventually if we only display the timestamps in the past X time window, we might want to consider using fixed values for some tests to check that old exceptions get weeded out (can't remember if this was something we discussed as being on the to-do list, or something like limiting by only keeping X most recent exceptions)

Reply:
That's NYI (not-yet-implemented). And the event source wouldn't understand any of that so it wouldn't be part of this test.

Clearing the cache from the startup hook exception pipeline step wouldn't be something observable by the dotnet-monitor side of things other than some type of reset event being emitted; it would only really be seen as the IDs being "reused".